sections/docker/image-tags.md

```diff
@@ -0,0 +1,29 @@
+# Understand image tags vs digests, and use the "latest" tag with caution
+
+### One Paragraph Explainer
+
+If this is a production situation, and security and stability are important, then just "convenience" is likely not the best deciding factor.
```

Comment:
I might be missing the point, but I think the part about `Understand[ing] image tags vs digests` is missing from the current explanation.

We could write that tagging is understandable by humans but risks being overwritten with a newer image while digests are immutable